src/material.cpp

```diff
@@ -1436,4 +1491,72 @@ openmc_extend_materials(int32_t n, int32_t* index_start, int32_t* index_end)
 
 extern "C" size_t n_materials() { return model::materials.size(); }
 
+//
+// cvmt functions 
+double 
+PolyProperty::evaluate(Position r) {
+  double results {0.0}; 
+  if(type_ == "zernike1d"){ 
+    results = evaluate_zernike1d(r);
+  } else if (type_ == "zernike"){
+    results = evaluate_zernike(r);
+   }
+  return results;
+}
+
+double 
+PolyProperty::evaluate_zernike1d(Position r) {
+  int i;
+  double rho;
+  int c_index;
+  double poly_val;
```

Comment:
Local variables should be declared / defined at the same time, and only when they are needed.